src/executors/qa-automation.yaml

```diff
@@ -0,0 +1,3 @@
+docker:
+  - image: codacy/qa-automation-test-runner:2.8.2
```

Comment:
Could we use the latest, instead of a specific version?
This is very dynamic, and this could bring another level every time we do a qa-automation-tests version. Or there's a way to have a default value as latest, and in case of need put the version as environment variable or something like that. What you think?

Reply:
I've made a commit to allow that to be configurable. I'll change the default to latest